src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js

```diff
@@ -0,0 +1,239 @@
+                      fieldName={fieldNames.lastname}
+                      id="lastname"
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.address}
+                      id="address"
+                      label={`*${_('checkoutProcessForm.contact_information_address')}`}
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                    <InputFieldItem
+                      type="text"
+                      label={`*${_('checkoutProcessForm.contact_information_city')}`}
+                      fieldName={fieldNames.city}
+                      id="city"
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.province}
+                      id="province"
+                      label={`*${_('checkoutProcessForm.contact_information_province')}`}
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                    <SelectFieldItem
+                      fieldName={fieldNames.country}
+                      id="country"
+                      label={`*${_('checkoutProcessForm.contact_information_country')}`}
+                      values={countries}
+                      required
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.zipCode}
+                      id="zipCode"
+                      label={`*${_('checkoutProcessForm.contact_information_zip_code')}`}
+                      withNameValidation
+                      className="field-item--50"
+                    />
+                    <PhoneFieldItem
+                      fieldName={fieldNames.phone}
+                      id="phone"
+                      label={`*${_('checkoutProcessForm.contact_information_phone')}`}
```

Comment:
What the asterisks (`*`) means?

Reply:
I mean, probably it represents required, in that case, it should be `FieldItem` responsibility.

@cbernat are we using asterisks (`*`) in other web app pages? (if not, we could keep this for a future PR)

Reply:
That the field is required:

![image](https://user-images.githubusercontent.com/70591946/124804913-31649900-df31-11eb-9302-f580e4718a8e.png)

Reply:
Ok, the current solution is simple, so, I think that it is fine keeping the asterisks for the moment, in the future we could improve it.

Reply:
yes, I agree with you